dockerfiles/jupyter/README.ipynb

```diff
@@ -8,29 +8,71 @@
     "\n",
-    "MLRun  backend service can run locally or over Kubernetes (preferred), see the instructions for installing it [**locally** using Docker](https://docs.mlrun.org/en/latest/install/local-docker.html) or [over **Kubernetes** Cluster](https://docs.mlrun.org/en/latest/install/kubernetes.html). Alternatively, you can use [Iguazio's **managed** MLRun service](https://www.iguazio.com/docs/latest-release/).\n",
+    "**For further details please check -**\n",
+    " * MLRun Documentation - [link](https://docs.mlrun.org/en/stable/index.html)\n",
+    "    * Projects Documentation - [link](https://docs.mlrun.org/en/stable/projects/project.html)\n",
+    "    * Functions Documentation - [link](https://docs.mlrun.org/en/stable/runtimes/functions.html)\n",
+    "    * Artifacts Documentation - [link](https://docs.mlrun.org/en/stable/store/artifacts.html)\n",
+    " * Quick start tutorial for machine learning tutorial - [link](https://docs.mlrun.org/en/stable/tutorials/01-mlrun-basics.html)\n",
+    " * MLRun cheat sheet - [link](https://docs.mlrun.org/en/stable/cheat-sheet.html#mlrun-setup)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ddeb6a5c",
+   "metadata": {},
+   "source": [
+    "### MLRun CE default components\n",
     "\n",
-    "This Jupyter Notebook server is designed to work with the different options, it works out of the box with the **local** deployment mode.<br> \n",
-    "In order to work with remote MLRun service (Kubernetes or managed) you need to edit and save the [**mlrun.env**](./mlrun.env) file in the following way:\n",
+    "The MLRun CE (Community Edition) chart includes the following components below by default:\n",
+    "* MLRun - https://github.com/mlrun/mlrun\n",
+    "  - MLRun API\n",
+    "  - MLRun UI\n",
+    "  - MLRun DB (MySQL)\n",
+    "* Nuclio - https://github.com/nuclio/nuclio\n",
+    "* Jupyter - https://github.com/jupyter/notebook (+MLRun integrated)\n",
+    "* MPI Operator - https://github.com/kubeflow/mpi-operator\n",
+    "* MinIO - https://github.com/minio/minio/tree/master/helm/minio\n",
+    "* Spark Operator - https://github.com/GoogleCloudPlatform/spark-on-k8s-operator\n",
+    "* Pipelines - https://github.com/kubeflow/pipelines\n",
+    "* Prometheus stack - https://github.com/prometheus-community/helm-charts\n",
+    "  - Prometheus\n",
+    "  - Grafana"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7aeb1ea8",
+   "metadata": {},
+   "source": [
+    "### MLRun CE Addional components \n",
     "\n",
-    "```sh\n",
-    "# set remote MLRun service address, username and access-key\n",
-    "MLRUN_DBPATH=https://<service-address>\n",
-    "V3IO_USERNAME=<user>\n",
-    "V3IO_ACCESS_KEY=<access-key>\n",
-    "```\n",
+    "There are a couple of default components enabled by default within the MLRun CE installation (see the list above). \n",
```

Comment:
"In addition to the default components, you can also enable the following components, if required: \n",